gradio/helpers.py

```diff
@@ -433,7 +438,8 @@ def __call__(
             total: estimated total number of steps.
             unit: unit of iterations.
         """
-        if self._active:
+        if self._callback:
+            assert self._callback is not None, "Callback not set."
```

Comment:
this assertion can never be reached?

Reply:
Oops meant to remove that, thanks for the catch